taichi/ui/backends/vulkan/renderable.cpp

```diff
@@ -144,7 +154,7 @@ void Renderable::create_graphics_pipeline() {
 void Renderable::create_vertex_buffer() {
   size_t buffer_size = sizeof(Vertex) * config_.max_vertices_count;
 
-  Device::AllocParams vb_params{buffer_size, false, false,
+  Device::AllocParams vb_params{buffer_size, true, true,
```

Comment:
May I ask why this is needed for Android? 
If this is really necessary, then we should guard it with `#if ANDROID` so that it doesn't hurt the performance of GGUI on other platforms.

Reply:
This is not needed, this is something I forget to revert during my testing... good catch! :D, it should be fixed.